app/controllers/compute_resources_controller.rb

```diff
@@ -112,13 +112,21 @@ def cluster_selected
     end
   end
 
+  def resource_pools
```

Comment:
Isn't this method equivalent to `api/v2/compute_resources/:id/available_clusters/:cluster_id/available_resource_pools"`?